### ISSUE TITLE
Games do not launch.

### ISSUE THREAD

**Issue by UnixTMDev** [ebenbruyns/junkstore#22]
Pretty much just what the title says. When I hit "Play Game", i just get kicked back out to the game list. Very notable problem. Not sure why it happens.

TL;DR: Play Game button sends me to the game list and never launches the game.

**Comment by jessebofill** [comment]
try usin a proton-ge version in the steam app page properties

**Comment by EldenG20** [comment]
Hmm I'm having an issue here where the game isn't being added to steam either, so I can neither launch the game from junk store nor set a different version of proton in steam.
On Beta branch of Steam OS

**Comment by ebenbruyns** [comment]
There's a new build that addresses this. It was a nasty bug with lots of race conditions hopefully this will be sorted now. Please retest and report back. 0.82 is the build you're looking for.

**Comment by EldenG20** [comment]
Thanks so much for such a fast resolution! Fixed it. 
Demons Tilt didn't work, but that might just be a deck issue as it didn't work through heroic launcher.
Got 20 minutes till dawn running straight away though.
Appears in library also.

**Comment by ebenbruyns** [comment]
yeah games can be a bit hit and miss, you can try proton GE 9-4 maybe that works. Standard rules applies for compatibility, but if it works in Heroic you should be able to get it running in Junk-Store. Both uses legendary under the covers.

**Comment by ebenbruyns** [comment]
Demons Tilt works if you run the exe from the sliders menu. This seems to be a rare case of the egs arguments screwing things up.

**Comment by ebenbruyns** [comment]
I found the issue, remove the \" and replace with just " around the executable in the shortcut's config. I think I've changed stuff that makes this no longer a requirement so I'll be testing over the next few days before I release a hotfix

**Comment by ebenbruyns** [comment]
If you want to help test you can join discord and I'll share a test build with you there

**Comment by EldenG20** [comment]
Thanks, just wrapping the executable in quotes fixed Demons Tilt, I know to look for that now, maybe other games have that same issue.

Discord link sounds great. Your solution is superior to Heroic in my view as it doesn't require the launching of a separate launcher, and Heroic kept logging me out of Epic randomly for reasons I've not yet discovered.

**Comment by EldenG20** [comment]
So yeah basically I'll probably be using this solution for epic games as my daily driver, happy help test out and report bugs.

**Comment by ebenbruyns** [comment]
That's very kind of you to say that, a lot of people's attitude towards this is "so what? heroic works", they're kinda missing out on the fact that it's extensible. Glad you're liking it, I hope it will just continue to improve from here on out.

**Comment by EldenG20** [comment]
This is less clicks than heroic too, and Heroics add to steam functionality requires some work. I

 think if you could get itch.io working that would be really interesting. 

Steam Deck is an awesome indie machine, though this might be a very niche use case. 

Happy to try and help contribute as I have a postgrad CS qualification, but little scripting experience. Bit time poor atm however.

**Comment by ebenbruyns** [comment]
Once you've done a bit of testing on the plugin as a whole, can you go write a test report on this PR: https://github.com/SteamDeckHomebrew/decky-plugin-database/pull/480

I need user feedback on there before it will be accepted into the stable store.
